sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/components.py

```diff
@@ -89,3 +89,14 @@ def eval_model(
     # Evaluate the incoming scoring result and output evaluation result.
     # Here only output a dummy file for demo.
     (Path(eval_output) / "eval_result").write_text("eval_result")
```

Comment:
instead of adding two functions, how about adding some primitive type output in exsiting function. 
like add a string output to Score and a string input to evaluate

Reply:
done.